plugins/nf-wave/src/main/io/seqera/wave/plugin/SubmitContainerTokenRequest.groovy

```diff
@@ -81,6 +81,11 @@ class SubmitContainerTokenRequest {
      */
     String spackFile
 
+    /**
+     * Target architecture for Spack builds
+     */
+    String spackArch
```

Comment:
Would not be better to call it `spackTarget` to stress that's a specific micro-architecture instead of the generic arch?

Reply:
Ok, can glob update it here and in the cousin PRs in Libseqera and Wave

Reply:
OK, moved to `spackTarget` in this PR, as well as in the related ones for Wave and Libseqera